app/components/NodeCircle.jsx

```diff
@@ -65,14 +65,24 @@ export default class NodeCircle extends BaseComponent {
          clip-path="${clipPath}">
        </image>`
     };
-
-    return image ? 
-      <g dangerouslySetInnerHTML={innerHTML} /> :
-      <circle 
+    if (image){
+      return <g dangerouslySetInnerHTML={innerHTML} />;
+    } else {
+      if (color){
```

Comment:
this can be simplified back to the previous version but with `color={color ? color : ds.circleColor[status]}` instead of `color={ds.circleColor[status]}`